completions/eog

```diff
@@ -19,7 +19,7 @@ _comp_cmd_eog()
         return
     fi
 
-    _comp_compgen_filedir '@(ani|avif|?(w)bmp|gif|ico|j2[ck]|jp[cefgx2]|jpeg|jpg2|pcx|p[bgp]m|pn[gm]|ras|svg?(z)|tga|tif?(f)|webp|x[bp]m)'
+    _comp_compgen_filedir '@(ani|avif|?(w)bmp|gif|heic|ico|j2[ck]|jp[cefgx2]|jpeg|jpg2|jxl|pcx|p[bgp]m|pn[gm]|ras|svg?(z)|tga|tif?(f)|webp|x[bp]m)'
```

Comment:
Should some more besides just `heic` be added for HEIC?

- https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format
- https://www.iana.org/assignments/media-types/image/heic

Reply:
Hmm good question.

What I am using is `heif-gdk-pixbuf`, which is based on [libheif](https://github.com/strukturag/libheif).<br>
It's not really clear to me, whether the different extension names are actually all different formats (okay some are HVEC based and some AV1) and whether libheif supports all of them.

OTOH, one could argue that these are simply well known extensions for image formats, `eog` is an image viewer and even if `heif-gdk-pixbuf` wouldn't support all formats, other loaders may and so it's still worth for bash-completion to recognise them.

If you wish, you can just extend my commit with the other extensions... or I could update it if you prefer.

Reply:
https://github.com/strukturag/libheif/blob/4a2f42f01e475258e67371db99108636d32f561c/gdk-pixbuf/pixbufloader-heif.c#L224-L229

Reply:
> It's not really clear to me, whether the different extension names are actually all different formats (okay some are HVEC based and some AV1) and whether libheif supports all of them.

Could you investigate them?

Reply:
Well I guess it's the one from the code you just referenced.

But as said before... do we even really want to include only those which are supported by `heif-gdk-pixbuf`? That could change any time, or there could be other GDK pixbuf loaders.

Reply:
> Well I guess

Could you confirm that?

> it's the one from the code you just referenced.

The code tells which extensions are at least supported, but it doesn't tell whether an extension not listed there is compatible with the listed one and could be loaded. If the file format is determined by only the filename extension, we should just list the extensions listed by the pixbuf implementation, but if the file format can be detected by other means (e.g. by checking several bytes at the beginning of the file), we can also list compatible extensions.

> But as said before... do we even really want to include only those which are supported by `heif-gdk-pixbuf`? That could change any time, or there could be other GDK pixbuf loaders.

We shouldn't include any extensions that are not known to be supported. Suggesting the file in the completion implies to the users that the file is supported by `eog`.